src/Core/Field.h

```diff
@@ -590,7 +590,7 @@ class Field
 
         switch (which)
         {
-            case Types::Null: return true;
```

Comment:
Here was an error - it incorrectly compared NEGATIVE_INFINITY and POSITIVE_INFINITY.